testdata/p4_16_samples_outputs/issue242-midend.p4

```diff
@@ -76,21 +76,33 @@ control Eg(inout Headers hdrs, inout Metadata meta, inout standard_metadata_t st
     bool tmp_2;
     bit<32> tmp_3;
     bit<32> tmp_4;
+    bool cond;
```

Comment:
Because the action contains method calls which previously we were ignoring, but we're now assuming might change/use any of the locals.  Strictly only needed if actions might contain nested actions (which I know we don't have any tests that do, but may be allowed by the syntax)